### ISSUE TITLE
Bad link from #cert_fault_tolerant_name_handling FAQ to 2.19 blog

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2555]
## Where to find the issue

- https://www.coronawarn.app/en/faq/#cert_fault_tolerant_name_handling "What happens if the names of my certificates do not exactly match?"
- https://www.coronawarn.app/de/faq/#cert_fault_tolerant_name_handling "Was passiert, wenn die Namen meiner Zertifikate nicht exakt übereinstimmen?"

## Describe the issue

The link to the 2.19 blog in https://www.coronawarn.app/en/faq/#cert_fault_tolerant_name_handling "What happens if the names of my certificates do not exactly match?" is incorrect. https://www.coronawarn.app/en/blog/2022-03-16-cwa-version-2-19/ does not exist, therefore clicking on the link in the following sentence produces a 
```
404 Sorry …
Something went wrong but maybe you will find something else interesting.
```
error.

"For more information, visit the blog: [CWA 2.19 improves the assignment of certificates to persons](https://www.coronawarn.app/en/blog/2022-03-16-cwa-version-2-19/)"

## Suggested change

Change the link in https://www.coronawarn.app/en/faq/#cert_fault_tolerant_name_handling "What happens if the names of my certificates do not exactly match?" to https://www.coronawarn.app/en/blog/2022-03-16-cwa-2-19/.

Similarly for the `/de` FAQ.

**Comment by MikeMcC399** [comment]
- PR #2556 submitted.

**Comment by MikeMcC399** [comment]
- This issue was resolved by PR #2556.

Check manually and with `npm test`.
